### ISSUE TITLE
Weird training results

### ISSUE THREAD

**Issue by KIMGEONUNG** [lllyasviel/ControlNet#206]
Hi, thank you for your great work.
I hava a question for the ControlNet training

I am just trying to train ControlNet with canny edge condition using OpenImage Dataset to reproduce or mimic your traning scheme. I preserve all training setting with the tutorial you gave us. However, the results is so weird like below and that was not recovered even after many training iterations. 

Did you have a experience like me during training? or is there any opinion to this phenomenon?
I just want to get some opinion or intuition about this from you.

Thank you.

The prompt is fixed by "a high-quality, detailed, and professional image"

**Input**
![gs-011025_e-000000_b-044100_control](https://user-images.githubusercontent.com/32098205/222352032-0cb52537-9360-482c-a38a-03f73fdb302f.png)

**Ourput**
![gs-011025_e-000000_b-044100_samples_cfg_scale_9 00](https://user-images.githubusercontent.com/32098205/222352212-d5124d7d-9b95-45b2-b017-bef0cd9853aa.png)

**Recon**
![gs-011025_e-000000_b-044100_reconstruction](https://user-images.githubusercontent.com/32098205/222352271-471f1f09-7dc4-4440-ab7a-f3e54d63d344.png)

**Comment by lllyasviel** [comment]
generate paired prompts and then use larger batch size (perhaps by accumulation)

**Comment by wenyihong** [comment]
> generate paired prompts and then use larger batch size (perhaps by accumulation)

What's the appropriate batch size, from your experience?

**Comment by kings-rgb** [comment]
> > generate paired prompts and then use larger batch size (perhaps by accumulation)
> 
> What's the appropriate batch size, from your experience?

Do you solve this problem?

**Comment by KIMGEONUNG** [comment]
> > > generate paired prompts and then use larger batch size (perhaps by accumulation)
> > 
> > 
> > What's the appropriate batch size, from your experience?
> 
> Do you solve this problem?

No, after many experiments, I tentatively concluded that the training code has unknown bug.
